doc-content/kogito-docs/src/main/asciidoc/openshift/chap-kogito-deploying-on-openshift.adoc

```diff
@@ -1230,12 +1230,170 @@ On Spring Boot, `spring.data.mongodb.uri` does not work simultaneously with host
 * {URL_CONFIGURING_KOGITO}#con-data-index-service_kogito-configuring[{PRODUCT} Data Index Service]
 * https://github.com/mongodb/mongodb-kubernetes-operator/blob/master/README.md[MongoDB Operator]
 
-////
+
 [id="proc-custom-kogito-service-integration-knative-eventing_{context}"]
 === Integrating a custom {PRODUCT} service with Knative Eventing
 
-<Start here>
+Knative Eventing is a serverless platform you can use to create event producers and consumers for your applications. For more information about Knative Eventing, see https://knative.dev/docs/eventing/[Knative Eventing] and the https://github.com/knative/docs[Knative docs repository] on GitHub.
+
+When you integrate a custom {PRODUCT} service with Knative Eventing you can create a Knative broker, bind that Knative broker with the {PRODUCT} Operator, and include a cross-reference link for Knative Eventing.
+
+Knative events conform to the CloudEvents specification. For more information, see https://cloudevents.io/[CloudEvents].
+
+You can use the Knative Eventing add-on to include your {PRODUCT} Services in the Knative Eventing platform. For more information about the Knative Eventing add-on, see https://github.com/kiegroup/kogito-runtimes/tree/main/addons/common/knative/eventing[Knative Eventing add-on] on GitHub.
+
+To use all features, in addition to those available as part of the {PRODUCT} Knative Eventing Source Operator, you must integrate the custom {PRODUCT} service with Knative Eventing.
+
+==== Creating a Knative Broker
+
+Knative brokers and triggers provide an "event mesh" model that enables an event producer to deliver events to a broker. The broker then uniformly distributes the events to consumers by using triggers. For more information about brokers, see https://knative.dev/docs/eventing/broker/[Broker - Knative]. For more information about triggers, see https://knative.dev/docs/eventing/broker/triggers/[Triggers - Knative].
+
+Consumers can register different types of events without needing to negotiate directly with event producers. You can use specific filter conditions to optimize event routing on the underlying platform.
+
+To integrate your custom {PRODUCT} Service with Knative Eventing you need at least one broker that collects the events produced by your custom {PRODUCT} Service. You can create a broker by applying a YAML file by using `kubectl`.
+
+.Prerequisites
+
+* Administrative privileges are available in the target cluster.
+* Knative Eventing is installed or the {OPENSHIFT} Serverless Platform is available.
++
+For more information about installing Knative Eventing, see https://knative.dev/docs/install/[About installing Knative]. For more information about {OPENSHIFT} Serverless, see https://www.openshift.com/learn/topics/serverless[{OPENSHIFT} Serverless]
+
+.Procedure
+
+. Create a `<filename.yaml>` YAML file containing the following template:
++
+----
+apiVersion: eventing.knative.dev/v1
```

Comment:
Just a small suggestion.
Can we use:
[SOURCE]
----
----
syntax here?

Reply:
Done